src/redux/Categories/categories.js

```diff
@@ -0,0 +1,15 @@
+const checkBookStatus = 'Bookstore/categories/checkStatus';
+
+const checkStatus = () => ({
+  type: checkBookStatus,
+});
+
+const categoryReducer = (state = [], action) => {
+  if (action.type === checkBookStatus) {
+    return 'Pending';
+  }
```

Comment:
- [ ] Kindly return the **'Under construction'** string instead of returning **'Pending'** which was a requirement in your project :+1: